tests/common/binder.py

```diff
@@ -84,17 +84,19 @@ def _generate_or_retrieve_user_manifest_v1(self, device):
     def force_user_manifest_v1_generation(self, device):
         self._generate_or_retrieve_user_manifest_v1(device)
 
-    def get_user_manifest_v1_for_device(self, device, ciphered=False):
+    def get_user_manifest_v1_for_device(
+        self, device: LocalDevice, ciphered=False
```

Comment:
`ciphered: bool` ?

Reply:
By inspecting the code that use this method, `ciphered` is never provided and since it isn't used I've removed it.